Gandheesh/1.Basics.html

```diff
@@ -0,0 +1,62 @@
+<html>
+
+<head>
+     <meta charset="UTF-8">
+     <meta name="description" content="Html basics">
+     <meta name="viewport" content="width=device-width,intial-scale=1.0">
+     <title> HTML Basics </title>
+</head>
+
+<body>
+     <h1> Merlo J. Pusey</h1>
+     <hr>
+     <p><cite> Wikipedia, the free encyclopedia</cite>
+     </p>
+     <img src="P1.jpg" width="400px" height="300px">
+     <p><b> Merlo John Pusey</b>(February 3, 1902, in Woodruff, Utah â€“ November 22, 1985, in Washington,
+          D.C.)<sup>[1][2]</sup> was an American biographer and editorial writer. He won the 1952 Pulitzer Prize for
+          Biography or Autobiography and the 1952 Bancroft Prize for his 1951 biography of U.S. Chief Justice Charles Evans Hughes.
+          <br><br>
+          Born and raised on a farm near Woodruff, Utah, Pusey was a Latter-day Saint. He attended the Latter-day Saints
+          University and graduated as a member of Phi Beta Kappa from the University of Utah after working on the
+          college newspaper. He later became
+          a reporter and assistant city editor at The Deseret News in Salt Lake City.[3]
+          <br><br>
+          <strike>Pussy</strike>Pusey worked for The Washington Post from 1928 to 1971, becoming associate editor in
+          1946, continuing to contribute occasional pieces until about two years before his death.</p>
+     <br><br>
+     <p><b>Contents</b>
+          <br>
+          
+          <ol>
+               <li><b><a href="#Books">Books</a></b></li>
+               <li><b><a href="#References">References</a></b></li>
+          </ol>
+     </p>
+     <br>
+     <br>
+     <h2><a name="Books">Books</h2>
+     <hr>
+     <p>
+          <ul type="disc">
+               <li>The Supreme Court Crisis(Macmillan, 1937)</li><br>
+               <li>Big Government: Can We Control It? (Harper, 1945)</li><br>
+               <li>Charles Evans Hughes (2 vols., Macmillan, 1951)</li><br>
+               <li> Eisenhower, the President (Macmillan, 1956)</li><br>
+     </p>
+     </ul>
+     <h2><a name="References">References</h2>
+     <hr>
+     <p>
+          <ul type="disc">
+               <li>Smith, J. Y. (1985-11-23). "Merlo J. Pusey Dies at 83". The Washington Post. ISSN 0190-8286.
+                    Retrieved 2016-03-24.
+               </li><br>
+               <li>Upi (1985-11-24). "Merlo J. Pusey Dies; Justice's Biographer Won a Pulitzer Prize". The New York
+                    Times. ISSN 0362-4331. Retrieved 2016-03-24.
+               </li><br>
+               <li>Latter-day Saint / Mormon Pulitzer Prize Winners". www.adherents.com. Retrieved 2016-03-24.</li><br>
+     </p>
```

Comment:
`p` tag is not needed. Use `div` with `id` References instead. `ul` tag is not closed. `br` tags are not needed in lists.